src/components/AssetsDropZone/index.jsx

```diff
@@ -79,7 +91,12 @@ export default class AssetsDropZone extends React.Component {
                   }}
                 >
                   { displayText => (
-                    <p aria-hidden id="asset-drop-zone-max-file-size-label" data-identifier="asset-drop-zone-max-file-size-label">
+                    <p
+                      className={styles['center-text']}
```

Comment:
Nit: you could alphabetize these attributes.

Reply:
Easy :) can do!